File: client/src/Types/Message.ts
export type Message = {
  id: string
  senderId: string
  senderDisplayName: string
  senderImageUrl: string
  recipientId: string
  recipientDisplayName: string
  recipientImageUrl: string
  content: string
  dateRead?: any
  messageSend: string
  currentUserSender?:boolean
}

File: API/Entities/Message.cs
using System;

namespace API.Entities;

public class Message
{
    public string Id { get; set; } = Guid.NewGuid().ToString();

    public required string Content { get; set; }

    public DateTime? DateRead { get; set; }

    public DateTime MessageSend { get; set; } = DateTime.UtcNow;

    public bool SenderDeleted { get; set; }

    public bool RecipientDeleted { get; set; }

    //nav property


    public required string SenderId { get; set; }

    public Member Sender { get; set; } = null!;

    public required string RecipientId { get; set; }

    public Member Recipient { get; set; } = null!;



}
